src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -52,7 +50,7 @@ def run_pf_command(*args, cwd=None):
 
 
 @pytest.mark.usefixtures(
-    "use_secrets_config_file", "setup_local_connection", "install_custom_tool_pkg", "recording_injection"
+    "use_secrets_config_file", "recording_injection", "setup_local_connection", "install_custom_tool_pkg"
```

Comment:
if so, we might need some comments to highligh this, otherwise we cannot promise its order for all devs

Reply:
I'll add some comment on recording injection method.

Reply:
Sry, this is no longer related. In previous version, I hijacked the Connection class, so that recording injection should come before setup local connection.
Now I removed the connection part so there is no sequence enforcement here.